gradlew

```diff
@@ -154,11 +154,19 @@ if $cygwin ; then
     esac
 fi
 
-# Split up the JVM_OPTS And GRADLE_OPTS values into an array, following the shell quoting and substitution rules
-function splitJvmOpts() {
-    JVM_OPTS=("$@")
+# Escape application args
+save ( ) {
```

Comment:
Not sure if this could lead to an issue - looks like there's an empty space b/w the parentheses.

Reply:
Oh, that's part of the Gradle wrapper, I haven't updated the code myself (graddle did for me :D )